lightning_pose/utils/scripts.py

```diff
@@ -498,6 +496,7 @@ def get_callbacks(
             monitor=None,
             every_n_epochs=ckpt_every_n_epochs,
             save_top_k=-1,
+            enable_version_counter=False,
```

Comment:
Minor fix. Previously if you didn't change hydra.run.dir, it would write out the model to version_0, then version_1... but we only ever load version 0. This flag disables that version incrementing behavior.